src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -315,11 +320,54 @@ private void createMovementLeftImage(BufferedImage image, String curMovement, St
     textG2D.setFont(new Font("Dialog", Font.BOLD, 20));
     final int textThicknessOffset = textG2D.getFontMetrics().stringWidth(curMovement) / 2;
     final boolean distanceTooBig = maxMovement.equals("");
-    textG2D.drawString(curMovement, distanceTooBig ? image.getWidth() / 2 - textThicknessOffset : 10, image.getHeight());
+    textG2D.drawString(curMovement, distanceTooBig ? image.getWidth() / 2 - textThicknessOffset : 10,
+        image.getHeight());
     if (!distanceTooBig) {
       textG2D.setColor(new Color(33, 0, 127));
       textG2D.setFont(new Font("Dialog", Font.BOLD, 16));
       textG2D.drawString(maxMovement, 10, image.getHeight());
     }
   }
+
+  /**
+   * Creates an Arrow-Shape
+   * 
+   * @param from The {@linkplain Point2D} specifying the direction of the Arrow
+   * @param to The {@linkplain Point2D} where the arrow is placed
+   * @return A transformed Arrow-Shape
+   */
+  private Shape createArrowTipShape(Point2D from, Point2D to) {
+    final Polygon arrowPolygon = new Polygon();
+    arrowPolygon.addPoint(-3, 2);
+    arrowPolygon.addPoint(1, 0);
+    arrowPolygon.addPoint(-3, -2);
+
+
+    final AffineTransform transform = new AffineTransform();
+    transform.translate(to.getX(), to.getY());
+    final double scale = 4;
+    transform.scale(scale, scale);
+    final double rotate = Math.atan2(to.getY() - from.getY(), to.getX() - from.getX());
+    transform.rotate(rotate);
+
+    return transform.createTransformedShape(arrowPolygon);
+  }
+
+  /**
+   * Draws an Arrow on the {@linkplain Graphics2D} Object
+   * 
+   * @param graphics The {@linkplain Graphics2D} object to draw on
+   * @param from The destination {@linkplain Point2D} form the Arrow
+   * @param to The placement {@linkplain Point2D} for the Arrow
+   * @param xOffset The horizontal pixel-difference between the frame and the Map
+   * @param yOffset The vertical pixel-difference between the frame and the Map
+   * @param scale The scale-factor of the Map
+   */
+  private void drawArrow(Graphics2D graphics, Point2D from, Point2D to, int xOffset, int yOffset, double scale) {
```

Comment:
you're not hitting any local state, looks like you can mark these private methods as static (note, static on private methods is a good thing since it makes the method easier to understand, provided you do not modify any static state. Static on public methods is bad, creates static coupling that hurts cohesion and makes testing difficult)